browser.js

```diff
@@ -13,7 +13,14 @@ const urls = {
 	v6: [
 		'https://ipv6.icanhazip.com/',
 		'https://api6.ipify.org/'
-	]
+	],
+	// These URLs accept both IPv6/IPv4 connectivity
+	// Response might change for dual IPv6/IPv4 stack network interface
+	// URL => regex which match IP in response
+	general: {
+		'https://www.cloudflare.com/cdn-cgi/trace': 'ip(=)(.*?)\n',
```

Comment:
```suggestion
		'https://www.cloudflare.com/cdn-cgi/trace': 'ip(=)(.*?)',
```
The body is trimmed anyway.

Reply:
We should not duplicate all this information. Move the shared info into a file that is imported by both the browser and Node.js implementation.

Reply:
Use named regex groups.